src/query/interpreter.cpp

```diff
@@ -791,6 +791,33 @@ Callback HandleStreamQuery(StreamQuery *stream_query, const Parameters &paramete
   }
 }
 
+Callback HandleConfigQuery() {
```

Comment:
Currently, this includes the `license_key`. Is this an issue?

Reply:
Currently we return all 56 start-up flags. Used to be 57, 4 of them used to be hidden we unhide 3 of them and deleted one.

Since the license and the org. name is allready available through `SHOW DATABASE SETTINGS;` I am guessing they should be made hidden. Is there anything else we would like to hide from the user? 

| **"auth_module_create_missing_role"**                                                                                                                                                                                                          | "Set to false to disable creation of missing roles."                                                                                                                                                                                                   |
| **"auth_module_create_missing_user"**                                                                                                                                                                                                                      | "Set to false to disable creation of missing users."                                                                                                                                                                                                   |
| **"auth_module_executable"**                                                                                                                                                                                                                               | "Absolute path to the auth module executable that should be used."                                                                                                                                                                                     |
| **"auth_module_manage_roles"**                                                                                                                                                                                                                             | "Set to false to disable management of roles through the auth module."                                                                                                                                                                                 |
| **"auth_module_timeout_ms"**                                                                                                                                                                                                                               | "Timeout (in milliseconds) used when waiting for a response from the auth module."                                                                                                                                                                     |
| **"auth_password_permit_null"**                                                                                                                                                                                                                            | "Set to false to disable null passwords."                                                                                                                                                                                                              |
| **"auth_password_strength_regex"**                                                                                                                                                                                                                         | "The regular expression that should be used to match the entire entered password to ensure its strength."                                                                                                                                              |
| **"allow_load_csv"**                                                                                                                                                                                                                                       | "Controls whether LOAD CSV clause is allowed in queries."                                                                                                                                                                                              |
| **"also_log_to_stderr"**                                                                                                                                                                                                                                   | "Log messages go to stderr in addition to logfiles"                                                                                                                                                                                                    |
| **"audit_buffer_flush_interval_ms"**                                                                                                                                                                                                                       | "Interval (in milliseconds) used for flushing the audit log buffer."                                                                                                                                                                                   |
| **"audit_buffer_size"**                                                                                                                                                                                                                                    | "Maximum number of items in the audit log buffer."                                                                                                                                                                                                     |
| **"audit_enabled"**                                                                                                                                                                                                                                        | "Set to true to enable audit logging."                                                                                                                                                                                                                 |
| **"auth_user_or_role_name_regex"**                                                                                                                                                                                                                         | "Set to the regular expression that each user or role name must fulfill."                                                                                                                                                                              |
| **"bolt_address"**                                                                                                                                                                                                                                         | "IP address on which the Bolt server should listen."                                                                                                                                                                                                   |
| **"bolt_cert_file"**                                                                                                                                                                                                                                       | "Certificate file which should be used for the Bolt server."                                                                                                                                                                                           |
| **"bolt_key_file"**                                                                                                                                                                                                                                        | "Key file which should be used for the Bolt server."                                                                                                                                                                                                   |
| **"bolt_num_workers"**                                                                                                                                                                                                                                     | "Number of workers used by the Bolt server. By default, this will be the number of processing units available on the machine."                                                                                                                         |
| **"bolt_port"**                                                                                                                                                                                                                                            | "Port on which the Bolt server should listen."                                                                                                                                                                                                         |
| **"bolt_server_name_for_init"**                                                                                                                                                                                                                            | "Server name which the database should send to the client in the Bolt INIT message."                                                                                                                                                                   |
| **"bolt_session_inactivity_timeout"**                                                                                                                                                                                                                      | "Time in seconds after which inactive Bolt sessions will be closed."                                                                                                                                                                                   |
| **"data_directory"**                                                                                                                                                                                                                                       | "Path to directory in which to save all permanent data."                                                                                                                                                                                               |
| **"isolation_level"**                                                                                                                                                                                                                                      | "Default isolation level used for the transactions. Allowed values: SNAPSHOT_ISOLATION, READ_COMMITTED, READ_UNCOMMITTED"                                                                                                                              |
| **"kafka_bootstrap_servers"**                                                                                                                                                                                                                              | "List of default Kafka brokers as a comma separated list of broker host or host:port."                                                                                                                                                                 |
| **"license_key"**                                                                                                                                                                                                                                          | "License key for Memgraph Enterprise."                                                                                                                                                                                                                 |
| **"log_file"**                                                                                                                                                                                                                                             | "Path to where the log should be stored."                                                                                                                                                                                                              |
| **"log_level"**                                                                                                                                                                                                                                            | "Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL"                                                                                                                                                                      |
| **"memory_limit"**                                                                                                                                                                                                                                         | "Total memory limit in MiB. Set to 0 to use the default values which are 100% of the phyisical memory if the swap is enabled and 90% of the physical memory otherwise."                                                                                |
| **"memory_warning_threshold"**                                                                                                                                                                                                                             | "Memory warning threshold, in MB. If Memgraph detects there is less available RAM it will log a warning. Set to 0 to disable."                                                                                                                         |
| **"monitoring_address"**                                                                                                                                                                                                                                   | "IP address on which the websocket server for Memgraph monitoring should listen."                                                                                                                                                                      |
| **"monitoring_port"**                                                                                                                                                                                                                                      | "Port on which the websocket server for Memgraph monitoring should listen."                                                                                                                                                                            |
| **"organization_name"**                                                                                                                                                                                                                                    | "Organization name."                                                                                                                                                                                                                                   |
| **"pulsar_service_url"**                                                                                                                                                                                                                                   | "Default URL used while connecting to Pulsar brokers."                                                                                                                                                                                                 |
| **"query_execution_timeout_sec"**                                                                                                                                                                                                                          | "Maximum allowed query execution time. Queries exceeding this limit will be aborted. Value of 0 means no limit."                                                                                                                                       |
| **"query_modules_directory"**                                                                                                                                                                                                                              | "Directory where modules with custom query procedures are stored. NOTE: Multiple comma-separated directories can be defined."                                                                                                                          |
| **"replication_replica_check_frequency_sec"**                                                                                                                                                                                                              | "The time duration between two replica checks/pings. If < 1, replicas will NOT be checked at all. NOTE: The MAIN instance allocates a new thread for each REPLICA."                                                                                    |
| **"storage_gc_cycle_sec"**                                                                                                                                                                                                                                 | "Storage garbage collector interval (in seconds)."                                                                                                                                                                                                     |
| **"storage_properties_on_edges"**                                                                                                                                                                                                                          | "Controls whether edges have properties."                                                                                                                                                                                                              |
| **"storage_recover_on_startup"**                                                                                                                                                                                                                           | "Controls whether the storage recovers persisted data on startup."                                                                                                                                                                                     |
| **"storage_restore_replicas_on_startup"**                                                                                                                                                                                                                  | "Controls replicas should be restored automatically."                                                                                                                                                                                                  |
| **"storage_snapshot_interval_sec"**                                                                                                                                                                                                                        | "Storage snapshot creation interval (in seconds). Set to 0 to disable periodic snapshot creation."                                                                                                                                                     |
| **"storage_snapshot_on_exit"**                                                                                                                                                                                                                             | "Controls whether the storage creates another snapshot on exit."                                                                                                                                                                                       |
| **"storage_snapshot_retention_count"**                                                                                                                                                                                                                     | "The number of snapshots that should always be kept."                                                                                                                                                                                                  |
| **"storage_wal_enabled"**                                                                                                                                                                                                                                  | "Controls whether the storage uses write-ahead-logging. To enable WAL periodic snapshots must be enabled."                                                                                                                                             |
| **"storage_wal_file_flush_every_n_tx"**                                                                                                                                                                                                                    | "Issue a \'fsync\' call after this amount of transactions are written to the WAL file. Set to 1 for fully synchronous operation."                                                                                                                      |
| **"storage_wal_file_size_kib"**                                                                                                                                                                                                                            | "Minimum file size of each WAL file."                                                                                                                                                                                                                  |
| **"stream_transaction_conflict_retries"**                                                                                                                                                                                                                  | "Number of times to retry when a stream transformation fails to commit because of conflicting transactions"                                                                                                                                            |
| **"stream_transaction_retry_interval"**                                                                                                                                                                                                                    | "Retry interval in milliseconds when a stream transformation fails to commit because of conflicting transactions"                                                                                                                                      |
| **"telemetry_enabled"**                                                                                                                                                                                                                                    | "Set to true to enable telemetry. We collect information about the running system (CPU and memory information) and information about the database runtime (vertex and edge counts and resource usage) to allow for easier improvement of the product." |
| **"query_cost_planner"**                                                                                                                                                                                                                                   | "Use the cost-estimating query planner."                                                                                                                                                                                                               |
| **"query_plan_cache_ttl"**                                                                                                                                                                                                                                 | "Time to live for cached query plans, in seconds."                                                                                                                                                                                                     |
| **"query_vertex_count_to_expand_existing"**                                                                                                                                                                                                                | "Maximum count of indexed vertices which provoke indexed lookup and then expand to existing, instead of a regular expand. Default is 10, to turn off use -1."                                                                                          |
| **"query_max_plans"**                                                                                                                                                                                                                                      | "Maximum number of generated plans for a query."                                                                                                                                                                                                       |
| **"flag_file"**                                                                                                                                                                                                                                            | "load flags from file"                                                                                                                                                                                                                                 |
| **"help"**                                                                                                                                                                                                                                                 | "show help on all flags [tip: all flags can have two dashes]"                                                                                                                                                                                          |
| **"help_xml"**                                                                                                                                                                                                                                             | "produce an xml version of help"                                                                                                                                                                                                                       |
| **"version"**                                                                                                                                                                                                                                              | "show version and build info and exit"                                                                                                                                                                                                                 |

Reply:
This looks good